manifests/base/commit.env

```
@@ -18,3 +18,11 @@ odh-rstudio-notebook-image-commit-n=af2a765
 odh-rstudio-notebook-image-commit-n-1=9fe149c
 odh-rstudio-gpu-notebook-image-commit-n=af2a765
 odh-rstudio-gpu-notebook-image-commit-n-1=9fe149c
+odh-codeserver-notebook-image-commit-n=a6c251e
+odh-codeserver-notebook-image-commit-n-1=6a6f888
+odh-rstudio-notebook-image-commit-n=a6c251e
+odh-rstudio-notebook-image-commit-n-1=6a6f888
+odh-rstudio-gpu-notebook-image-commit-n=a6c251e
+odh-rstudio-gpu-notebook-image-commit-n-1=6a6f888
+odh-rocm-pytorch-notebook-image-commit-n=
+odh-rocm-pytorch-notebook-image-commit-n=
```

```suggestion
odh-rocm-tensorflow-notebook-image-commit-n=
```
one of this should be tensorflow